src/entities/Customer.java

```diff
@@ -185,6 +188,8 @@ private String makeBooking(ArrayList<Cinema> cinemaDB, ArrayList<Movie> movieDB)
 				}
 			}
 			String TID = Integer.toString(cinema.getName().hashCode()%1000) + showtime.toString();
+			
+			//Constructor has been updated, creation of ticket here need to match constructor
 			Ticket newTicket = new Ticket(mov, cinema, this, showtime, cinema.getSeat(title+showtime.toString(), index), TID, this.username, this.email, this.mobileNumber);
```

Comment:
I've made the edits based on your comments for other classes, but since you wrote Ticket, could you make the edit here and push another commit reflecting that?